src/core/singletons/bootstrap.ts

```diff
@@ -194,6 +196,127 @@ export class CoreBootstrap {
         });
     }
 
+    /**
+     * Handle Bootstrap tab elements in a certain element.
+     * Supports both Bootstrap 4 and 5.
+     * https://getbootstrap.com/docs/5.3/components/navs-tabs/#tabs
+     *
+     * @param rootElement Element where to search for elements to treat.
+     */
+    protected static handleTabs(rootElement: HTMLElement): void {
+        const elements = Array.from(rootElement.querySelectorAll<HTMLElement>(
+            '.list-group, .nav, [role="tablist"]',
+        ));
+
+        if (!elements.length) {
+            return;
+        }
+
+        elements.forEach((element) => {
+            if (!element.hasAttribute('role')) {
+                element.setAttribute('role', 'tablist');
+            }
+            const children = Array.from(element.querySelectorAll<HTMLElement>('.nav-link:not(.dropdown-toggle),\
+                .list-group-item:not(.dropdown-toggle),\
+                [role="tab"]:not(.dropdown-toggle),\
+                [data-bs-toggle="tab"],[data-toggle="tab"],\
+                [data-bs-toggle="pill"],[data-toggle="pill"],\
+                [data-bs-toggle="list"],[data-toggle="list"]'));
```

Comment:
This code doesn't follow our coding guidelines in terms of wrapping. Also, I guess the string will end up with several blank spaces between each option (because of the indentation), which probably don't cause any problem but it can look weird. It might be better to use an array with all the options and then do `.join(,)`.

In this same function you have a couple more cases where you're using a multiline string in querySelector. Those are ok in terms of coding guidelines, so they can stay as they are if they work, but they can also be converted to array for consistency if you prefer. As you wish.

If you prefer to keep this one as a multiline string, then it should be like this:


```suggestion
            const children = Array.from(element.querySelectorAll<HTMLElement>(
                '.nav-link:not(.dropdown-toggle),\
                .list-group-item:not(.dropdown-toggle),\
                [role="tab"]:not(.dropdown-toggle),\
                [data-bs-toggle="tab"],[data-toggle="tab"],\
                [data-bs-toggle="pill"],[data-toggle="pill"],\
                [data-bs-toggle="list"],[data-toggle="list"]',
            ));
```

Reply:
I've changed all of them a bit not to repeat them so much.